pkgs/by-name/pc/pcsx2-bin/update.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq gnused common-updater-scripts
+
+set -eou pipefail
+
+version="$(curl --silent "https://api.github.com/repos/PCSX2/pcsx2/releases" | jq '.[0].tag_name' --raw-output | sed 's/^v//')"
+
+if [ "$version" == "null" ]; then
+    echo "error: could not fetch pcsx2 latest version from GitHub API" >&2
+    exit 1
+fi
+
+update-source-version pcsx2-bin "$version"
```

Comment:
Usually we test against the current version. Does this function do this?

Reply:
Yes apparently, by adding the `--ignore-same-version` argument to it